packaging/resource_suite_s3_nocache.py

```diff
@@ -1910,6 +1910,39 @@ def test_istream_nonexistent_file__issue_6479(self):
                 'DATA_REPL_STATUS', '0'])
             self.user0.run_icommand(['irm', '-f', logical_path])
 
+    def test_istream_append__issue_2153(self):
+        replica_number_in_s3 = 0
+        filename = 'test_istream_append__issue_2153.txt'
+        logical_path = os.path.join(self.user0.session_collection, filename)
+        content = 'this is the original file for istream'
+        content_append1 = 'line1 appended to the file'
+        content_append2 = 'line2 appended to the file'
+
+        try:
+            # istream to a new data object and ensure that it is created successfully.
+            self.user0.assert_icommand(['istream', 'write', logical_path], input=content)
+            self.assertEqual(str(1), lib.get_replica_status(self.user0, filename, replica_number_in_s3))
+
+            # append to the file
+            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'\n{content_append1}')
+
+            # append to the file again
+            self.user0.assert_icommand(['istream', 'write', '-a', logical_path], input=f'\n{content_append2}')
```

Comment:
The reasons I did it that way is because of the assertion in 1936.  If I put the newline in those it searches for lines that have the newline character in it and fails.

If there is a better way to approach this then I would be happy to make a change.

Reply:
Kory pointed out that using 'STDOUT' vs 'STDOUT_MULTILINE' handles the newline issue so I made the change so the vars now have the newline in it.